src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-mount-details/cephfs-mount-details.component.ts

```diff
@@ -10,23 +14,77 @@ import { BaseModal } from 'carbon-components-angular';
 export class CephfsMountDetailsComponent extends BaseModal implements OnInit {
   @ViewChild('mountDetailsTpl', { static: true })
   mountDetailsTpl: any;
+  mountDetail!: Record<string, any>;
   onCancel?: Function;
   private MOUNT_DIRECTORY = '<MOUNT_DIRECTORY>';
-  mountData!: Record<string, any>;
-  constructor(public activeModal: NgbActiveModal) {
-    super();
-  }
+  filteredUsers: any[] = [];
+  selectedUser: string = 'client.admin';
+  updatedUser: any;
```

Comment:
lets avoid using `any` and use typings, same for other places.
Thanks

Reply:
Changed any to specific type.